code/modules/recycling/conveyor2.dm

```diff
@@ -320,6 +320,8 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 
 /// Called when a user clicks on this switch with an open hand.
 /obj/machinery/conveyor_switch/interact(mob/user)
```

Comment:
attack ghost is calling can_use(user), and if the test passes, it's call interact.

so this is living check should be in can_interact